common/services/app/civic-uk/scripts.tsx

```diff
@@ -1,3 +1,38 @@
+import { font } from '@weco/common/utils/classnames';
+
+const branding = {
+  removeIcon: true,
```

Comment:
Icon is removed, but decide with Jason where the cookie preference centre link should be and look like.